src/main/java/org/sonar/plugins/resharper/ReSharperSensor.java

```diff
@@ -170,7 +172,7 @@ private static void logSkippedIssueOutsideOfSonarQube(ReSharperIssue issue, File
   }
 
   private static void logSkippedIssue(ReSharperIssue issue, String reason) {
-    LOG.info("Skipping the ReSharper issue at line " + issue.reportLine() + " " + reason);
+    LOG.debug("Skipping the ReSharper issue at line " + issue.reportLine() + " " + reason);
```

Comment:
You are right, with exclusions, this message would be more meaningful. I haven't tested this in a larger environment yet, only used the example project.